src/main/java/coffeetime/config/ObjectStorageConfig.java

```diff
@@ -0,0 +1,43 @@
+package coffeetime.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.S3Configuration;
+
+import java.net.URI;
+
+@Configuration
+public class ObjectStorageConfig {
+
+    @Value("${oci.object-storage.namespace}")
+    private String namespace;
+
+	@Value("${oci.object-storage.region}")
+	private String region;
+
+    @Value("${oci.object-storage.access-key}")
+    private String accessKey;
+
+    @Value("${oci.object-storage.secret-key}")
+    private String secretKey;
+
+	@Bean
+	public S3Client s3Client() {
+		return S3Client.builder()
+                .region(Region.of(region))
+                .credentialsProvider(StaticCredentialsProvider.create(
+                        AwsBasicCredentials.create(accessKey, secretKey)
+                ))
+                .endpointOverride(URI.create("https://" + namespace + ".compat.objectstorage." + region + ".oraclecloud.com"))
```

Comment:
여기는 크게 문제 없는데, + 연산이 많은 문자열 합성은 실수하기 쉬워서, `formatted`를 쓰면 좋을 것 같습니다.
```suggestion
                .endpointOverride(URI.create("https://%s.compat.objectstorage.%s.oraclecloud.com".formatted(namespace, region))
```

Reply:
앗 생각을 못했어요 반영할게요 감사합니다 ~~! 오 덕분에 commit suggestion 첨써봤어요 [1ee9e56](https://github.com/coffeetimeteam/coffeetime-server/pull/49/commits/1ee9e561bd9d48736ec66ce14df018b37bef3d4c)으로 반영됐습니다~!